heartbeat/findif.sh

```diff
@@ -233,24 +233,19 @@ findif()
     fi
     case $1 in
     */*) : OK ;;
+    # "ip route" doesnt show netmask for IPv6 /128
+    *:*:*) : OK ;;
     *)
       ocf_log err "Unable to find cidr_netmask."
       return $OCF_ERR_GENERIC ;;
     esac
   fi
   [ -z "$nic" ] && nic=$3
   [ -z "$netmask" ] && netmask=${1#*/}
-  if [ $family = "inet" ] ; then
-    if [ -z "$brdcast" ] ; then
-      if [ -n "$7" ] ; then
-        set -- `ip -o -f $family addr show | grep $7`
-        [ "$5" = brd ] && brdcast=$6
-      fi
-    fi
-  else
-    if [ -z "$OCF_RESKEY_nic" -a "$netmask" != "${1#*/}" ] ; then
-      ocf_log err "Unable to find nic, or netmask mismatch."
-      return $OCF_ERR_GENERIC
+  if [ -z "$brdcast" ] ; then
+    if [ -n "$7" ] ; then
+      set -- `ip -o -f $family addr show | grep $7`
+      [ "$5" = brd ] && brdcast=$6
```

Comment:
This part is not necessary to run for inet6 because ```brd``` is only shown when inet.
It would not be harm for the results but it would add a wasteful invocation of the ip command when inet6.